### ISSUE TITLE
Using in standard browser

### ISSUE THREAD

**Issue by luigi37** [jzeiders/gps-filter#1]
Hi, first of all thanks for having developed this library!
I would like to use it in a javascript application (without node).
Is it possible? How?
Sorry for the specific question which is not strictly related to the project but I think it might interest others.
Thanks

**Comment by jzeiders** [comment]
Hey so I just added a file called browser_filter. It was created using Browserify and when included in a project will bind gpsFilter to the window where it can be used globally.

**Comment by luigi37** [comment]
Thanks a lot!

~~Meantime I think I found a similar solution having learnt about browserify.
Only I wanted to expose the functions so that I could use them without having to use browserify again...
I added into my fork the file:
https://github.com/luigi37/gps-filter/blob/master/gpsfilter.js
with the following functions:
window.GPSpositionFilter = function (points,min,max)
window.GPSvelocityFilter = function (points,min,max)
window.GPSaccelerationFilter = function (points,min,max)
window.GPSremoveSpikes = function (points,sharpness,iterations)
window.GPSsmoothLine = function (points,threshold)
which mimic yours.
Having created them with "window" enables their usage in other scripts loaded after.
Feel free to add to your repo.~~

~~BTW: I've found something strange in acceleration filter:
newPoints=GPSaccelerationFilter(newPoints,0,30); //9.3 is from 0 to 100km/h in 3 sec - 5,6 is in 5 sec---
I assume that min and max are expressed in m/s2
Actually I recorded a track during a bycicle trip.
I've set min 0 and max 9.3 (which is like a Formula 1 acceleration if I did not make mistakes in calculation)
The log is quite precise but I've got many points (70%) removed...
I had to increase to 30 to include almost all the points but I think it's weird (too high acc)
Did I miss anything?~~

Thanks
Luigi
